targets/REV3-SGM/main.cpp

```diff
@@ -1,30 +1,147 @@
-/**
- * This is a basic sample of using the UART module. The program provides a
- * basic echo functionality where the uart will write back whatever the user
- * enters.
- */
-
+#include <EVT/dev/MCUTimer.hpp>
```

Comment:
Add a comment at the top of this file saying that it's the main target for the bike